homeassistant/components/tts/legacy.py

```diff
@@ -227,6 +227,11 @@ def supported_options(self) -> list[str] | None:
         """Return a list of supported options like voice, emotions."""
         return None
 
+    @callback
+    def async_get_supported_voices(self, language: str) -> list[str] | None:
```

Comment:
What will call this method?

Reply:
It will be called by the WS API, maybe we should add that also in this PR?